ci/README.md

```diff
@@ -211,6 +211,26 @@ Once a job has been removed from the required list, it requires two
 consecutive successful nightly test runs before being made required
 again.
 
+### Maintainership
+
+Tests should transition to required status within a reasonable timeframe after
+being added/updated. Without this progression, tests face several challenges
+that can impact the overall quality of the project:
```

Comment:
As discussed last week, in my opinion there is a place for non-required test for "experimental/unsupported" platforms/features. The trade off for this is that there is no needs for the community to ensure that test pass for these, which can lead to regressions as you discuss below, but I'm not sure that we want to rule out having these. I'm happy to hear from the wide community though.

Reply:
Yeah, I will make a test categorized more specifically. Thanks.